packages/now-static-build/src/index.ts

```diff
@@ -17,6 +17,7 @@ import {
   BuildOptions,
   Config,
 } from '@now/build-utils';
+import isPortReachable from "is-port-reachable"
```

Comment:
Use single quotes for consistency.

```suggestion
import isPortReachable from 'is-port-reachable'
```